ui/tests/integration/components/sync/secrets/page/destinations/destination/secrets-test.js

```diff
@@ -84,5 +86,55 @@ module(
       await click(menu.unsync);
       await click(PAGE.confirmButton);
     });
+
+    module('flash messages', function (hooks) {
+      hooks.beforeEach(function () {
+        const flashMessages = this.owner.lookup('service:flash-messages');
+
+        this.flashSuccessSpy = sinon.spy(flashMessages, 'success');
+        this.flashDangerSpy = sinon.spy(flashMessages, 'danger');
+      });
+
+      test('unsync should render flash messages', async function (assert) {
+        await click(PAGE.menuTrigger);
+
+        const { menu } = PAGE.associations.list;
+        await click(menu.unsync);
+        await click(PAGE.confirmButton);
+
+        assert.true(
+          this.flashSuccessSpy.calledWith('Unsync operation initiated.'),
+          'Success message is displayed'
+        );
+        assert.true(this.flashDangerSpy.notCalled);
+      });
+
+      test('sync now should render flash messages', async function (assert) {
+        await click(PAGE.menuTrigger);
+
+        const { menu } = PAGE.associations.list;
+        await click(menu.sync);
+
+        assert.true(
+          this.flashSuccessSpy.calledWith('Sync operation initiated.'),
+          'Success message is displayed'
+        );
+        assert.true(this.flashDangerSpy.notCalled);
+      });
+
+      test('it should show an error message when sync fails', async function (assert) {
+        this.server.post('/sys/sync/destinations/:type/:name/associations/set', () => {
+          return new Response(500);
+        });
+
+        await click(PAGE.menuTrigger);
+
+        const { menu } = PAGE.associations.list;
+        await click(menu.sync);
+
+        assert.true(this.flashSuccessSpy.notCalled);
```

Comment:
do these assertions just return `true` or `false`? If that's the case, do we want to include a message to explain what was or wasn't called?